pkg/jobrunaggregator/jobrunaggregatoranalyzer/cmd.go

```diff
@@ -51,6 +54,7 @@ func (f *JobRunsAnalyzerFlags) BindFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&f.ExplicitGCSPrefix, "explicit-gcs-prefix", f.ExplicitGCSPrefix, "only used by per PR payload promotion jobs.  This overrides the well-known mapping and becomes the required prefix for the GCS query")
 	fs.DurationVar(&f.Timeout, "timeout", f.Timeout, "Time to wait for aggregation to complete.")
 	fs.StringVar(&f.EstimatedJobStartTimeString, "job-start-time", f.EstimatedJobStartTimeString, fmt.Sprintf("Start time in RFC822Z: %s", kubeTimeSerializationLayout))
+	fs.StringVar(&f.JobStateQuerySource, "query-source", jobrunaggregatorlib.JobStateQuerySourceBigQuery, "The source from which job states are found. It is either bigquery or cluster")
```

Comment:
The two implementations are for different analyzers. One is for regular and PR aggregator, while the other is for test case analyzer (e.g. install). They are different enough to have different cmd.go.

Reply:
I see, could you get that spelled out very clearly in the option and/or godoc.

Why does is test case analyzer not able to use a similar approach of watching prowjobs? I'm glad Payload and PR Payload both get the new though.

Reply:
I might have confused you. Test case analyzer IS supported in this PR. The jobrunaggregatoranalyzer/cmd.go is for both payload and PR aggregator since they both have very similar functionality. The jobruntestcaseanalyzer/cmd.go is for test case analyzer. The command options and the logic are different enough to warrant a separate implementation. 

In this case, both analyzers have the same command argument "query-source". Thus this is added in both cmd.go files.